eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java

```diff
@@ -265,6 +269,11 @@ private MutableListMultimapFactory()
                 fastListMultimap.put(key3, value3);
                 return fastListMultimap;
             }
+
+            public <K, V> MutableListMultimap<K, V> withAll(Multimap<K, V> multimap)
+            {
+                return new FastListMultimap(multimap);
```

Comment:
In addition, I think the generics of the parameter should be `Multimap<? extends K, ? extends V> multimap` to match the generics on the constructor that the method delegates to.